### PR TITLE
ArtisanHomePage activity and UserHomePage activity have been changed …

### PR DESCRIPTION
**Pull request by shrinidhi99**
…along with two additional permissions added to android Manifest

### CODE REVIEW

File: ArtisansFinal/.idea/dictionaries/shrinidhi.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="shrinidhi">
```
Comment (rohit-mp) [inline ArtisansFinal/.idea/dictionaries/shrinidhi.xml]: Why is this file here?

File: ArtisansFinal/build.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.3.1'
```
Comment (rohit-mp) [inline ArtisansFinal/build.gradle]: please pull from upstream before submitting a pr.. the current version is ```3.2.1``` to help use a library and so let's keep it at that

File: ArtisansFinal/.idea/dictionaries/shrinidhi.xml (hunk 1 above)
Comment (shrinidhi99) [reply ArtisansFinal/.idea/dictionaries/shrinidhi.xml]: i have finally found the file. Two ways of removing it. One is go to the gitignore file of the project and add a line of code there which looks the best option. Another option is to delete the file from the project folder.
which one do you recommend?

Comment (shrinidhi99) [reply ArtisansFinal/.idea/dictionaries/shrinidhi.xml]: /.idea/dictionaries.xml
this is the line to be added
